plugins/woocommerce-blocks/assets/js/blocks/mini-cart/style.scss

```diff
@@ -2,6 +2,10 @@
 	display: inline-block;
 }
 
+.wp-block-woocommerce-mini-cart {
+	font-size: var(--wp--preset--font-size--medium);
```

Comment:
I think that we should provide a fallback value, in case `--wp--preset--font-size--medium` isn't defined.

This is how Gutenberg does: https://github.com/WordPress/gutenberg/blob/c20ad9fb2be3cab6e749b1193140acedcb80d81a/packages/block-library/src/post-comments-form/style.scss#L81-L82